Tests/PlotTests/Assertions.swift

```diff
@@ -10,11 +10,12 @@ import Plot
 func assertEqualHTMLContent(
     _ document: HTML,
     _ content: String,
+    _ expected: String? = nil,
     file: StaticString = #file,
     line: UInt = #line
 ) {
     let html = document.render()
-    let expectedPrefix = "<!DOCTYPE html><html>"
+    let expectedPrefix = expected ?? "<!DOCTYPE html><html>"
```

Comment:
What's the purpose of this change?

Reply:
That was a modification for the HTML5 Boilerplate test, `.html()` has `.class()` and `.lang()` attributes which renders to "`<!DOCTYPE html><html class="no-js" lang="en">`" so a closed "`<html>`" would cause failure.  This was just a minimal method (and effort) to allow for customization.  Perhaps the "`<html>`" should be removed from the `expectedPrefix` test?  Some variation could be useful still because it is additive, defaults to nil and allows for testing attributes on the "`<html>`" element.

Reply:
Perhaps additionally something like this to HTMLTests.swift

```
    func testPageClass() {
        let html = HTML(.class("no-js"))
        XCTAssertEqual(html.render(), #"<!DOCTYPE html><html class="no-js"></html>"#)
    }

    func testPageLanguageClass() {
        let html = HTML(.class("no-js"),.lang(.english))
        XCTAssertEqual(html.render(), #"<!DOCTYPE html><html class="no-js" lang="en"></html>"#)
    }
```